setup.py

```diff
@@ -14,9 +14,12 @@
 
 from __future__ import print_function
 import os
+from os import path
+from codecs import open
 from setuptools import setup
 import versioneer
 
+here = path.abspath(path.dirname(__file__))
```

Comment:
Duplicate logic/lines for grabbing the `README` path vs. the `requirements.txt`?